java/build-android.gradle

```diff
@@ -160,6 +173,16 @@ publishing {
 						id = 'onnxruntime'
 						name = 'ONNX Runtime'
 						email = 'onnxruntime@microsoft.com'
+                    }
+                }
+
+            	if (qnnVersion != null) {
```

Comment:
nit: These lines are indented with spaces but the rest of the file is using tabs.